src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.operations.BalanceOperationHandler;
+import core.basesyntax.operations.PurchaseOperationHandler;
+import core.basesyntax.operations.ReturnOperationHandler;
+import core.basesyntax.operations.SupplyOperationHandler;
+import core.basesyntax.service.FileService;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.TransactionParser;
+import core.basesyntax.service.impl.FileServiceImpl;
+import core.basesyntax.service.impl.OperationStrategyImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.TransactionParserImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+
+    private static final String INPUT_FILE_PATH = "src/main/java/resources/inputdata.csv";
```

Comment:
Place the input and output files into the **src/main/resources** folder.
https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop#place-the-input-and-output-files-into-the-srcmainresources-folder